### PR TITLE
Adding environment variable for marionette port forwarding

### PR DESCRIPTION
**Pull request by eliperelman**

### CODE REVIEW

File: src/main/java/jenkins/plugins/testdroid/DeviceSessionWrapper.java [hunk 1]
```diff
@@ -219,6 +220,7 @@ public void buildEnvVars(Map<String, String> env) {
                 env.put("ANDROID_SERIAL", adbJSONObject.getString("serialId"));
                 env.put("MARIONETTE_PORT", marionetteJSONObject.getString("port"));
                 env.put("MARIONETTE_HOST", host);
+                env.put("MARIONETTE_FORWARD_PORT", marionetteJSONObject.getString("forwardPort"))
```
Comment (davehunt) [inline src/main/java/jenkins/plugins/testdroid/DeviceSessionWrapper.java]: Missing `;` at the end of the line. I'll fix this locally and test your patch. If it works, I'll go ahead and merge.

**Comment by davehunt** [comment]
System log:
```
Testdroid adb proxy response: [{
    "forwardPort": 5037,
    "forwardHost": "172.27.240.149",
    "sessionId": "75147",
    "type": "adb",
    "serialId": "f03a405c",
    "port": 1028,
    "createdAt": "2015-08-03T11:24:55.632Z",
    "updatedAt": "2015-08-03T11:24:55.632Z",
    "id": 955}]
Testdroid marionette proxy response: [{
    "forwardPort": 30011,
    "forwardHost": "172.27.240.149",
    "sessionId": "75147",
    "type": "marionette",
    "port": 1029,
    "createdAt": "2015-08-03T11:24:55.705Z",
    "updatedAt": "2015-08-03T11:24:55.705Z",
    "id": 956}]
```

Job console:
```
[Testdroid] - ADB port: 1028
[Testdroid] - ADB host: fxos.testdroid.com
[Testdroid] - Android serial: f03a405c
[Testdroid] - Marionette port: 1029
[Testdroid] - Marionette host: fxos.testdroid.com
[Testdroid] - Marionette forwarding port: 30011
```

So that looks good, but I wonder if we also need an `ADB_FORWARD_PORT` environment variable? Would we need to restore this after an `adb reboot` too? Also, is there anything we might need the forward hosts for? /cc @srautiai @eliperelman

**Comment by eliperelman** [comment]
I'm not sure about the forward host, but it sounds like it would be needed.

**Comment by davehunt** [comment]
@JohanLorenzo do you want to pick this up? The change looks good to me. You'll need to run `mvn clean package` to build the plugin and you can test it in a local Jenkins instance. Once we're happy with it we can upload it to our Jenkins instance and restart for the changes to take effect.

**Comment by JohanLorenzo** [comment]
I tried the logs locally. I got something similar to @davehunt : 
```
[Testdroid] - Marionette forwarding host: 172.27.240.144
[Testdroid] - Marionette forwarding port: 30000
```

I'm okay with merging the patch.

**Comment by JohanLorenzo** [comment]
@eliperelman The package has been deployed on our Jenkins instance. As an example, see: http://jenkins1.qa.scl3.mozilla.com/view/UI/job/flame-kk-319.b2g-inbound.tinderbox.ui.functional.sanity.bitbar/2633/console

**Pull request merged**